config/channel/distributed/200-controller-clusterrolebinding.yaml

```diff
@@ -27,3 +27,20 @@ roleRef:
   kind: ClusterRole
   name: eventing-kafka-channel-controller
   apiGroup: rbac.authorization.k8s.io
+
+---
+
```

Comment:
Some places have space between the YAML dividers (e.g. config/channel/webhook/webhook-configuration.yaml) and some do not (e.g. config/channel/webhook/webhook-hpa.yaml).  Is there a more-proper way ?

Reply:
Seems about 50/50 for all the YAML in eventing-kafka - not inclined to bother with aligning them all as part of this work.  Might be a standardization win for the future though - not sure it's that important ; )